docs/DeveloperGuide.md

```diff
@@ -112,6 +112,10 @@ Named Arguments:
 --ignore-standalone-config, -i
                      A flag to ignore the standalone config file in
                      the repo.
+--timezone ZONE_ID[±hh[mm]], -t ZONE_ID[±hh[mm]]
+                     The timezone to use for the generated report. 
+                     The timezone should be recognized by Java ZoneId class. E.g. UTC
```

Comment:
give a few more examples. An example should be _typical_; the one given is a special case.
The user is supposed to know ZoneId class? Let's know leak implementation details to the user. Is there any other way to specify the acceptable values?

Reply:
The zone-id class takes in four kinds of zone-ids which cannot be easily described in the UG or DG.
What I can do is give more examples. However, I would like to suggest the users use the UTC[±hh[mm]] because it is easier to use.

Reply:
It is also acceptable to ask the users to use UTC only.

Reply:
Perhaps instead of the second line `The timezone should be recognized by Java ZoneId class.`,
we can list some out for the users for e.g. `Example of ZONE_ID: UTC, GMT etc recognized by Java ZoneId class.`

Reply:
Although four formats work, we don't have to explain all four. Just describe one? The other three can be a _hidden_ feature.

Reply:
@damithc Can you review the latest message?